### ISSUE TITLE
Valet not found 404-Not Found

### ISSUE THREAD

**Issue by Gardezi1** [cpriego/valet-linux#64]
Hello, guys last night I turned off my laptop without turning off the valet service. Now I'm facing a problem. Whenever I try to hit blog.dev or the URL given with it I'm getting a 404 error 

![image](https://cloud.githubusercontent.com/assets/13199724/25606990/dc091646-2f2d-11e7-85c6-a338ad587765.png)

![image](https://cloud.githubusercontent.com/assets/13199724/25607076/5180911a-2f2e-11e7-8dd2-af1f643b5268.png)

Please tell me what is it that is going wrong. I even pinged blog.dev to see if it was hitting 127.0.0.1 and yeah it's hitting it 

![image](https://cloud.githubusercontent.com/assets/13199724/25607008/f81ddc40-2f2d-11e7-846c-3515708b7786.png)

**Comment by cpriego** [comment]
Please use `valet paths` to see if your folders are "parked" in valet.

**Comment by Gardezi1** [comment]
@cpriego Yes my directory has been added to valet path. Here is the output `This directory has been added to Valet's paths.`

![image](https://cloud.githubusercontent.com/assets/13199724/25621016/94601790-2f69-11e7-968e-3d62792e6b8e.png)

**Comment by cpriego** [comment]
Are nginx and php-fpm running?
 `valet status`

**Comment by Gardezi1** [comment]
@cpriego yes they both are runnning 

![image](https://cloud.githubusercontent.com/assets/13199724/25633169/b3a035d8-2f8f-11e7-9ca9-17b89e3b3ef5.png)

**Comment by Gardezi1** [comment]
@cpriego  can you please tell me what is going wrong.

**Comment by cpriego** [comment]
OK, I think I see what could the problems be:
1. Make sure the directory `/var/www/html` is owned by your user (it seems is `rohan`)
2. Unpark/forget the `/var/www/html/blog` folder with `valet forget`
3. Park **only** the `/var/www/html` folder (it seems there's where your projects live)

Just to be sure show me the output of the commands:
- `ls -lh` and
- `valet paths`

**Comment by Gardezi1** [comment]
on which directory do you want to see this output

**Comment by Gardezi1** [comment]
Here is the output 
![image](https://cloud.githubusercontent.com/assets/13199724/25633852/f8d7b71e-2f91-11e7-97d0-dd8dbb7ae04a.png)

**Comment by Gardezi1** [comment]
![image](https://cloud.githubusercontent.com/assets/13199724/25634061/b520811c-2f92-11e7-85c3-46abaf848781.png)

**Comment by Gardezi1** [comment]
Well, I do not know what was going wrong. What I did is I stopped my Nginx server and started my apache server and then  tried valet serve and then tried to access the URL and it is working 

I do not know what was going wrong but if you can shed some light on this issue then that would be a great help. 

Thanks :)

**Comment by cpriego** [comment]
What's wrong is you're parking the wrong folder. You must park only the `/var/www/html`

**Comment by cpriego** [comment]
@Gardezi1 is the problem solved?

**Comment by Gardezi1** [comment]
@cpriego Sorry for the late reply. Was busy with some stuff. Will be updating it today 

:smile_cat:

**Comment by Gardezi1** [comment]
@cpriego  yeah the problem is solved. Closing the issue

**Comment by hakamibrahim** [comment]
when I enter the url "blog.dev" there is a description "This site can not be reached". but when i ping the url "blog.dev" its status reply.
![screenshot from 2018-01-24 22-12-00](https://user-images.githubusercontent.com/18589069/35339664-c1fe9b9c-0153-11e8-95fd-57c97788546d.png)

**Comment by hakamibrahim** [comment]
this is what the browser displays when accessing the url @cpriego @Gardezi1 
![screenshot from 2018-01-24 22-12-12](https://user-images.githubusercontent.com/18589069/35339905-5e11b7c6-0154-11e8-9838-f9255e443ac2.png)

**Comment by TheNodi** [comment]
Google Chrome doesn't work well with `.dev` domains anymore. 
Switch to `.test` using `valet domain test` and try to access `http://blog.test`.

Ping will always respond, since every domain ending with the configured TLD will point to localhost. `127.0.53.53` is a fancy response saying you have some conflicts in DNS resolution (since `.dev` is a registered TLD by Google) but in the end it will point to localhost as well.

Also, valet serves sites using your linux user, be sure to have at least read access on `/var/www/html`. It would be better to switch to something inside your home directory, like `/home/user/sites`.

**Comment by hakamibrahim** [comment]
Ok thanks,now it works for me :) @TheNodi

**Comment by EricHier** [comment]
Hi! I've got the same problem: nginx respons with an 404 - Not Found when visiting the page. So far I've already tried everything on this page as well as reinstalling. I will give you the output of some commands: 

![Bildschirmfoto von 2020-03-02 16-10-00](https://user-images.githubusercontent.com/17355638/75688751-580d2280-5ca0-11ea-9d5b-2e6dace447cd.png)

What could be the problem?

**Comment by jmarcher** [comment]
@EricHier Please open a new issue, closed issues are hard to follow.

**Comment by besabellacyrus** [comment]
Ahh i get it, you must park the root folder not your laravel root folder, for me i created a development folder, then inside development folder i created a laravel project via laravel new laravel-sms, then after that, i cd back into the development folder and then do valet park. its working!!!
